compress_raw_reports_telemetry.sh

```diff
@@ -0,0 +1,16 @@
+#! /bin/bash
+
+# Deal with spaces in filenames:
+# https://www.reeltoreel.nl/wiki/index.php/Dealing_with_spaces_in_filenames
+# The default value of IFS is " \t\n" (e.g. <space><tab><newline>)
+SAVEIFS=$IFS
+IFS=$(echo -en "\n\b")
```

Comment:
I see it's in the article, but "\b" is really weird to me here; any idea why it's there?

Reply:
yes, I also found it weird, I'm not sure why they added backspace here